fbchat/graphql.py

```diff
@@ -169,7 +229,8 @@ def graphql_to_user(user):
         own_nickname=c_info.get('own_nickname'),
         photo=user['profile_picture'].get('uri'),
         name=user.get('name'),
-        message_count=user.get('messages_count')
+        message_count=user.get('messages_count'),
+        event_reminders=event_reminders
```

Comment:
To avoid longer diffs in the future, it's good practice to always append a trailing comma at the end of these definitions. Just a heads up 😉

Reply:
Okey, thanks. Done